UI/DisplayLayoutScreen.cpp

```diff
@@ -192,37 +191,87 @@ void DisplayLayoutScreen::CreateViews() {
 	horizontalBoundaryR->AddTab("", bottomBoundary);
 
 	Choice *back = new Choice(di->T("Back"), "", false, new AnchorLayoutParams(leftColumnWidth, WRAP_CONTENT, 10, NONE, NONE, 10));
-	static const char *zoomLevels[] = { "Auto", "1x", "2x", "3x", "4x", "5x", "6x", "7x", "8x", "9x", "10x" };
-	zoom_ = new PopupMultiChoice(&g_Config.iSmallDisplayZoom, gr->T("Zoom settings"), zoomLevels, 0, ARRAY_SIZE(zoomLevels), gr->GetName(), screenManager(), new AnchorLayoutParams(300, WRAP_CONTENT, verticalBoundaryPositionL * 2 - 150, NONE, NONE, 10));
-	zoom_->OnChoice.Handle(this, &DisplayLayoutScreen::OnZoomChange);
-
-	
-
+	static const char *zoomLevels[] = { "Stretching", "Partial Stretch", "Auto Scaling", "Manual Scaling" };
+	zoom_ = new PopupMultiChoice(&g_Config.iSmallDisplayZoomType, cw->T("Options"), zoomLevels, 0, ARRAY_SIZE(zoomLevels), gr->GetName(), screenManager(), new AnchorLayoutParams(400, WRAP_CONTENT, local_dp_xres / 2 - 200, NONE, NONE, 10));
```

Comment:
Hmm.  I'll defer to anyone doing translations, but it seems unsual to use `Options` from another section.

Maybe "Cheating Options" and "Stretching/Scaling Options" might be two separate words in some non-English language, like "High [quality]" and "Tall [display]" are different words in English.

-[Unknown]